### DISCUSSION TITLE
Example of Different card forms

### DISCUSSION THREAD

**Discussion by CalebWelch** [flutter-stripe/flutter_stripe#218] [General]
https://user-images.githubusercontent.com/14021660/126713847-8a7d14a3-49f5-49fc-9cdf-df4b0d2bf278.mp4

Are there any examples of modifying or editing the layout of the default CardField? The default UI is really annoying to use as you cant view all the fields at once, but I haven't found a way of modifying it to make it more readable.

**Comment by remonh87** [comment]
We use the standard Stripe elements because if you want to deviate from it your organisation needs to be PCI compliant. However Stripe added a much nicer and easier integration called the `Payment sheet`.   The amazing thing is that it looks much better, is fully localized and for you as developer easier to integrate.

That said we also will facilitate for integrating your own cardfield but make sure  that your organisation understands the impact of becoming PCI compliant: https://stripe.com/en-se/guides/pci-compliance.    .

**Reply by CalebWelch** [reply]
https://user-images.githubusercontent.com/14021660/127067176-beade660-3069-43eb-bd7c-0455dfa007b9.mp4

I'm trying to work with payment sheets, but the default example in the code base doesn't seem to work properly out of the box. I can't get the payment sheet to pop up or stay open for confirmation

**Reply by lbk9** [reply]
@remonh87 that is a dead link, can you update. It would be useful for my current project as the focus issues in the CardField on Android require the use of the CustomCardField approach in the example app.

**Comment by NeatFastro** [comment]
Can we change the default country of payment sheet from United States to other?
County code parameter does not seems to work. I have already changed it to 'NLD' but looks like it has no effect
